README.md

```diff
@@ -1,41 +1,28 @@
-# PCO Services API Developer Support
+# PCO API Developer Support
 
-Planning Center Services has an API that allows you to access most of the information about your plans, people, songs, arrangements, etc. Documentation on the available API endpoints is [available here](http://get.planningcenteronline.com/api).
+Planning Center has an API that allows you to access most of the information in your account, across multiple apps. Documentation on the available API endpoints is [available here](http://planningcenter.github.io/api-docs/).
 
 ## Authentication
 
-We use [OAuth 1.0a](http://oauth.net/core/1.0) to allow third party applications access to your data without using your password. OAuth can be tricky if you are new to it, but we have example applications below to get you started.  Keep in mind we currently support [PLAINTEXT](http://tools.ietf.org/html/rfc5849#section-3.4.4) method for OAuth signatures.
-
-Before integrating your app with PCO Services, you'll need to request API keys by emailing [support](mailto:support@planningcenteronline.com). Be sure to include a URL for your app or church in your request.
-
 ### Single User Authentication
 
-If your app only needs access to your own data, this is the easiest method for authentication. This is typically how apps that sync data to your ChMS work. In that case, OAuth is much easier to implement because you only need to obtain and reuse a single access token.
-
-Once you've received your consumer key and secret by emailing support, you can use [accesstoken.io](http://accesstoken.io) to generate an access token for whichever user you are currently logged in as.
+If your app only needs access to your own data, this is the easiest method for authentication. This is typically how apps that sync data to your ChMS work.
 
-Here are some apps showing how to use your access token:
+You can get 'Personal Access Tokens' [here](https://api.planningcenteronline.com/oauth/applications).  You can then use HTTP basic auth to access the API.
 
-- [Ruby](/examples/ruby-single-user)
-- [PHP (PECL Library)](/examples/php-single-user)
-- [PHP (OAuth Library)](/examples/php-single-user-oauthlib)
+```bash
+curl -u application_id:secret https://api.planningcenteronline.com/services/v2/
```

Comment:
Is `application_id` the "Client ID" listed here?

![image](https://cloud.githubusercontent.com/assets/2231765/9819097/df67faaa-5860-11e5-9e55-f08cc644eb98.png)


If so, I think we should use `client_id`, just so it matches the UI.